c7n/resources/rds.py

```diff
@@ -2226,8 +2226,8 @@ def process(self, resources, event=None):
 
 
 @filters.register('pending-maintenance')
-class PendingMaintenance(Filter):
-    """ Scan DB instances for those with pending maintenance
+class PendingMaintenance(ValueFilter):
```

Comment:
@sgungi-intuit Do we need to change this from Filter to ValueFilter? In the example you gave, I don't see you're using it as a ValueFilter. My understanding is that you're just adding action details as an annotation and then you use a separate ValueFilter against the annotations. We don't need PendingMaintenance to be a ValueFilter itself right?

Reply:
Reverted to "Filter" from "ValueFilter"